pkgs/development/python-modules/pkgconfig/default.nix

```diff
@@ -19,11 +19,9 @@ buildPythonPackage rec {
 
   checkInputs = [ nose ];
 
-  nativeBuildInputs = [ pkgconfig ];
+  propagatedNativeBuildInputs = [ pkgconfig ];
```

Comment:
Is this the non python pkg-config? Please rename this to have a dash.

Reply:
Yup, and will do.